libraries/exoplayer_dash/src/test/java/androidx/media3/exoplayer/dash/DefaultDashChunkSourceTest.java

```diff
@@ -249,9 +249,9 @@ public FallbackSelection getFallbackSelectionFor(
     }
     assertThat(Lists.transform(chunks, (chunk) -> chunk.dataSpec.uri.toString()))
         .containsExactly(
-            "http://video.com/baseUrl/a/video/video_0_700000.m4s",
             "http://video.com/baseUrl/a/video/video_0_452000.m4s",
             "http://video.com/baseUrl/a/video/video_0_250000.m4s",
+            "http://video.com/baseUrl/a/video/video_0_700000.m4s",
```

Comment:
This test errors a chunk download, excludes the respective track, picks another track and downloads the chunk in a loop until a replacement track is not available and then validates the order of the tracks.

As the initial network type was C.NETWORK_TYPE_UNKNOWN, It was picking a track closer to 700 Kbps (1 Mbps x 0.7 BANDWIDTH_FRACTION) and the order was 700 Kbps, 452 Kbps, 250 Kbps and 1.3 Mbps.

After the change, we use the default network type (2G) from robolectric for which exo's default bandwidth estimate is 574 Kbps (820Kbps x 0.7 BANDWIDTH_FRACTION) and so the order changed to 452 Kbps, 250 Kbps, 700 Kbps and 1.3 Mbps.

Reply:
Jfyi, Default Locale from robolectric is `en_US`.